src/libraries/Common/src/System/Net/HttpDateParser.cs

```diff
@@ -35,17 +35,23 @@ internal static class HttpDateParser
             "d MMM yyyy H:m:s", // RFC 5322 no day-of-week, no zone
         };
 
-        // Try the various date formats in the order listed above.
-        // We should accept a wide variety of common formats, but only output RFC 1123 style dates.
-        internal static bool TryStringToDate(ReadOnlySpan<char> input, out DateTimeOffset result) =>
-             DateTimeOffset.TryParseExact(
-                 input,
-                 s_dateFormats,
-                 DateTimeFormatInfo.InvariantInfo,
-                 DateTimeStyles.AllowWhiteSpaces | DateTimeStyles.AssumeUniversal,
-                 out result);
+        internal static bool TryStringToDate(ReadOnlySpan<char> input, out DateTimeOffset result)
```

Comment:
TryParseStringToDate?

Reply:
This was the existing name. But I changed it to just TryParse.